src/Service/HealthCheck/HealthCheckService.php

```diff
@@ -50,8 +56,24 @@ public function getHealthCheckResult(): HealthCheckResult
 
         $cacheStatistics = $resultCacheImplementation->getStats();
 
+        $connection = $this->entityManager->getConnection();
+
+        try {
+            $testQuery = $connection->getDatabasePlatform()->getDummySelectSQL();
+            $testQueryResult = (bool)$connection->fetchOne($testQuery);
+        } catch (Exception $e) {
```

Comment:
Maybe use `DBALException` or something more specific?

Reply:
Also please rename `$e` for something meaningful

Reply:
> Maybe use DBALException or something more specific?
That's already a specific exception from DBAL :) It just named like a "core" one. And DBALException itself is deprecated
> Also please rename $e for something meaningful
Will do, thx!